core/src/main/java/io/questdb/cairo/AbstractIntervalPartitionFrameCursor.java

```diff
@@ -194,22 +201,11 @@ private long computeSize() {
                 if (partitionTimestampLo >= intervalLo) {
                     lo = 0;
                 } else {
-                    lo = binarySearch(column, intervalLo, partitionLimit == -1 ? 0 : partitionLimit, rowCount, SCAN_UP);
-                    if (lo < 0) {
-                        lo = -lo - 1;
-                    }
-                }
-
-                long hi = binarySearch(column, intervalHi, lo, rowCount - 1, SCAN_DOWN);
-
-                if (hi < 0) {
-                    hi = -hi - 1;
-                } else {
-                    // We have direct hit. Interval is inclusive of edges and we have to
-                    // bump to high bound because it is non-inclusive
-                    hi++;
+                    lo = timestampFinder.findTimestamp(intervalLo - 1, partitionLimit == -1 ? 0 : partitionLimit, rowCount - 1) + 1;
```

Comment:
this looks like a regression, the original code is searching for "intervalLo" the new one is using "intervalLo - 1"

Reply:
It's not a regression. As discussed, I've added a few fuzz tests for time interval intrinsics in aa0502362767e7fcddeb46a03d72c8aae8bc7b33

PTAL

As for the `rowCount-1` value being used instead of the `rowCount` value as the upper binary search value, both lower and upper boundaries in binary search are inclusive, so the old code could access unmapped memory.